Runtime/DotNetE2EServerApp/packages.config

```diff
@@ -27,13 +27,14 @@
   <package id="Microsoft.Owin.Security.OAuth" version="2.1.0" targetFramework="net45" />
   <package id="Microsoft.Owin.Security.Twitter" version="2.1.0" targetFramework="net45" />
   <package id="Microsoft.WindowsAzure.ConfigurationManager" version="2.0.3" targetFramework="net45" />
-  <package id="Newtonsoft.Json" version="6.0.1" targetFramework="net45" />
+  <package id="Newtonsoft.Json" version="6.0.4" targetFramework="net45" />
   <package id="Owin" version="1.0" targetFramework="net45" />
   <package id="RazorEngine" version="3.4.1" targetFramework="net45" />
   <package id="System.IdentityModel.Tokens.Jwt" version="3.0.2" targetFramework="net45" />
   <package id="System.Spatial" version="5.6.0" targetFramework="net45" />
-  <package id="WindowsAzure.MobileServices.Backend" version="1.0.348" targetFramework="net45" />
+  <package id="WindowsAzure.MobileServices.Backend" version="1.0.450" targetFramework="net45" />
   <package id="WindowsAzure.MobileServices.Backend.Entity" version="1.0.348" targetFramework="net45" />

```

Comment:
Following packages still referring to 1.0.348:
WindowsAzure.MobileServices.Backend.Entity, WindowsAzure.MobileServices.Backend.Tables
They need to updated as well to 1.0.450